src/CanIHazHouze.AppHost/AppHost.cs

```diff
@@ -77,16 +78,23 @@
     .WithReference(ledgerService)       // ← Keep for cross-service verification
     .WithHttpHealthCheck("/health")
     .WaitFor(documentService)           
-    .WaitFor(ledgerService);            
+    .WaitFor(ledgerService);
+
+var crmService = builder.AddProject<Projects.CanIHazHouze_CrmService>("crmservice")

```

Comment:
probably this should go to the csporj file as well, to import the actual project.